.github/hub/push_evaluations_to_hub.py

```diff
@@ -98,8 +98,7 @@ def push_module_to_hub(module_path, type, token, commit_hash, tag=None):
 
 
 if __name__ == "__main__":
-    evaluation_paths = ["metrics", "comparisons", "measurements"]
-    evaluation_types = ["metric", "comparison", "measurement"]
+    evaluation_paths = ["suites"]
```

Comment:
Are we disabling those other 3 evaluations?

Reply:
We don't use these. Types were only used to distinguish different HF orgs on HF hub. I originally removed them so it would not try to add them for hugging face metrics. But it actually should not matter since we are using a different directory. We can add metrics back in a new PR when we have a metric to add.